src/sdk.js

```diff
@@ -536,6 +539,28 @@ class EventsSDK {
       }
     }
   }
+  _getTabsSession(){
+      if (!window.sessionStorage.length) {
+        // Ask other tabs for session storage
+        localStorage.setItem('getSessionStorage', Date.now());
+      };
+      window.addEventListener('storage', (event) => {
+        //console.log('storage event', event);
+        if (event.key == 'getSessionStorage') {
+          // Some tab asked for the sessionStorage -> send it
+          localStorage.setItem('sessionStorage', JSON.stringify(window.sessionStorage));
```

Comment:
Not sure if this is a good idea to store the sessionStorage object inside localStorage